src/core/data-tests.js

```diff
@@ -23,11 +23,50 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  const emojiList = document.createElement("span");
```

Comment:
Note, this should be an array really... otherwise, this could end up with either and empty span or with a nested set of spans:

```html
<a href="...">name-of-test</a> 
<!-- nested span are not great -->
<span>
   <span>💪</span>
</span>
```

We want to end up with markup like this:
```HTML
<li>
  <a href="...">name-of-test</a> <span>💪</span> <span>🔒</span>
</li>
```

Reply:
This is still remaining to be fixed, right? (i.e., `const emojiList = [];`, instead of using a span)